<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviewWidget</class>
 <widget class="QWidget" name="PreviewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prévisualisation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="comboLabel">
     <property name="text">
      <string>Fichiers à enregistrer</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="applyButton">
     <property name="text">
      <string>&amp;Apply</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="quitButton">
     <property name="text">
      <string>&amp;Quit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
